src/main/java/com/internship/frejaeidjmeterplugin/jmeter/sampler/impl/FrejaEidPluginSampler.java

```diff
@@ -37,19 +45,44 @@ public void setRequests(String requests) {
         setProperty("requests", requests);
     }
 
+    public String getEmailFilePath() {
```

Comment:
this should be remove , also this three methods bellow, i forgot that part of code.